File: frontend/src/app/dto/user/user.update.res.dto.ts
import { UserUpdateDataResDto } from "./user.update.data.res.dto"

export class UserUpdateResDto{
    private _data! : UserUpdateDataResDto
	private _message! : string

    public get data(): UserUpdateDataResDto {
        return this._data;
    }

    public set data(data: UserUpdateDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }
    
}

File: backend/src/main/java/com/lawencon/ticketms/dto/user/UserUpdateResDto.java
package com.lawencon.ticketms.dto.user;

public class UserUpdateResDto {
	private UserUpdateDataResDto data;
	private String message;
	public UserUpdateDataResDto getData() {
		return data;
	}
	public void setData(UserUpdateDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
